nservicebus/router/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Router
+summary: How to connect parts of the system that use different transports 
+component: Router
+related:
+- samples/azure/azure-service-bus-msmq-bridge
+- samples/msmq/sql-bridge
+reviewed: 2018-05-04
+---
+
+`NServiceBus.Router` is a universal component that connects parts of an NServiceBus-based solution that otherwise cannot talk to each other (e.g. because they use different transport or transport settings or because of physical barriers). 
+
+Unlike the [Gateway](/nservicebus/gateway/) or the [Wormhole](/nservicebus/wormhole/), the Router handles both sending and publishing. Unlike the [Bridge](/nservicebus/bridge/) the Router can use site-based addressing to route message between logically significant sites (just like the Gateway does).
+
+The Router is transparent to the publishing and replying endpoint. That is:
+
+ * The endpoint that *replies* to a message does not have to know if the initiating message came through a router. The reply will be routed automatically to the correct router and then forwarded to the initiating endpoint.
+ * The endpoint that *publishes* events does not have to know if the subscribers are behind the router.
+
+
+## Connecting to the router
+
+Regular endpoints connect to the router using *connectors* that allow them to configure the routing
+
+snippet: connector
+
+The snippet above tells the endpoint that a designated router listens on queue `MyRouter` and that messages of type `MyMessage` should be sent to the endpoint `Receiver` via the router. It also tell the subscription infrastructure that the event `MyEvent` is published by the endpoint `Publisher` that is hosted behind the router.
+
+
+## Router configuration
+
+The following snippet shows a simple MSMQ-to-RabbitMQ router configuration
```

Comment:
Maybe explain here how the router is hosted? It's unclear to a first-time reader if this is hosted on its own or inside an endpoint.